From 4fdfbfaf489edc45e6fc1fc6770789a10d96c0e8 Mon Sep 17 00:00:00 2001
From: Didier 'OdyX' Raboud <didier.raboud@liip.ch>
Date: Mon, 5 Feb 2018 17:19:03 +0100
Subject: Drop the promo blocks

---
 src/Template/Auth/login.ctp | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/src/Template/Auth/login.ctp b/src/Template/Auth/login.ctp
index f6ada67ec..95f542976 100644
--- a/src/Template/Auth/login.ctp
+++ b/src/Template/Auth/login.ctp
@@ -35,11 +35,7 @@ if ($browser == 'firefox' || $browser == 'chrome') {
         <?php echo $this->element($pluginCheckTemplate); ?>
     </div>
     <div class="row">
-        <div class="col3 push1 github-block">
-            <?php echo $this->element('Public/Promo/cloud'); ?>
-        </div>
-        <div class="col3 github-block">
-            <?php echo $this->element('Public/Promo/pro'); ?>
+        <div class="col6">
         </div>
         <div class="col4 github-block push1 last">
             <?php echo $this->element('Public/Promo/github'); ?>
