--- a/app/Config/core.php.default
+++ b/app/Config/core.php.default
@@ -24,7 +24,7 @@
 /**
  * Add the composer autoloader script.
  */
-require_once dirname(__DIR__) . '/Vendor/autoload.php';
+require_once '/usr/share/php/passbolt-api/app/Vendor/autoload.php';
 
 /**
  * CakePHP Debug Level:
--- a/app/Config/bootstrap.php
+++ b/app/Config/bootstrap.php
@@ -123,7 +123,10 @@
 	CakePlugin::load('DataTests');
 }
 
-require_once dirname(dirname(__DIR__)) . DS . 'Vendor' . DS . 'autoload.php';
+// Only load if the healtchecks-needed libraries is installed
+if (file_exists('/usr/share/php/passbolt-api/Vendor/autoload.php')) {
+    require_once '/usr/share/php/passbolt-api/Vendor/autoload.php';
+}
 
 /**
  * Plugins need to be loaded manually, you can either load them one by one or all of them in a single call
--- a/app/Config/app.php.default
+++ b/app/Config/app.php.default
@@ -47,7 +47,7 @@
 			'setenv' => true,
 			// otherwise you can set the location here
 			// typically on Centos it would be in '/usr/share/httpd/.gnupg'
-			'home' => '/home/www-data/.gnupg',
+			'home' => '/var/lib/passbolt/gnupghome',
 		],
 		// Main server key
 		'serverKey' => [
