--- a/app/View/Users/register.ctp
+++ b/app/View/Users/register.ctp
@@ -13,12 +13,11 @@
 <div class="grid">
 	<div class="row">
 		<div class="col6 push1 information">
-			<h2><?php echo __('Try passbolt demo today!'); ?></h2>
+			<h2><?php echo __('Get an account on this passbolt instance now!'); ?></h2>
 			<p>
 				<?php echo __('Enter your details in the form.'); ?>
 				<?php echo __('We will send you an email to get you started.'); ?>
 			</p>
-<?php echo $this->element('public/disclaimer-legal'); ?>
 		</div>
 		<div class="col4 push1 last">
 			<div class="logo">
@@ -32,8 +31,7 @@
 					<?php echo $this->Form->input('Profile.last_name', array('label' => __('Last name'), 'class' =>' fluid')); ?>
 					<?php echo $this->Form->input('User.username', array('label' => __('Email'), 'class' =>' fluid')); ?>
 					<p>
-					<input type="checkbox" name="disclaimer" id="disclaimer" value="value" required="required">
-					<label for="disclaimer" style="font-size:.9em"><?php echo __('I understand the disclaimer. I agree with the Terms of Service and Privacy Policy.'); ?></label>
+					<input type="hidden" name="disclaimer" id="disclaimer" value="1">
 					</p>
 				</fieldset>
 				<div class="submit-wrapper">
@@ -44,4 +42,4 @@
 			</div>
 		</div>
 	</div>
-</div>
\ No newline at end of file
+</div>
--- a/app/View/Users/register_thankyou.ctp
+++ b/app/View/Users/register_thankyou.ctp
@@ -14,9 +14,8 @@
 		<div class="col6 push1 information">
 			<h2><?php echo __('Thank you'); ?></h2>
 			<p>
-				<?php echo __('Thank you for giving passbolt a test run! We just sent you an email, please follow the instructions to get started.'); ?>
+				<?php echo __('Thank you for subscribing to this passbolt instance! We just sent you an email, please follow the instructions to get started.'); ?>
 			</p>
-<?php echo $this->element('public/disclaimer-legal'); ?>
 		</div>
 		<div class="col4 push1 last">
 			<div class="logo">
